src/chainparams.h

```diff
@@ -150,3 +150,8 @@ inline unsigned int GetMinimumConnectionsRequiredForStaking()
 {
     return fTestNet ? 1 : 3;
 }
+
+inline unsigned int GetMaxInputsForConsolidationTxn()
+{
+    return (unsigned int) 600;
+}
```

Comment:
Protocol-wise, there's nothing enforcing a certain max amount of inputs so this function shouldn't be in chainparams I think. (policy maybe?)

On a related note now that I think about it, this also applies to `GetMinimumConnectionsRequiredForStaking`.

Reply:
You are right that both of those are not enforced on the receive side per se. Neither one are in the consensus data structure, which is appropriate.

Reply:
Policy.h/cpp seems better.

Reply:
I will move both of those to there I think. Good recommendation.